lib/classes/record/adapter.php

```diff
@@ -566,6 +566,8 @@ public function get_duration()
     public function move_to_collection(collection $collection, appbox $appbox = null)
     {
         if ($this->getCollection()->get_base_id() === $collection->get_base_id()) {
+            $this->dispatch(RecordEvents::COLLECTION_CHANGED, new CollectionChangedEvent($this, $collection, $collection));
+
```

Comment:
possibly bad side effects with code (plugins ?) that listen to **real** collection changes.
we must fix the (lost) indexation pb where it happens (e.g. posting a reindex evt)